build-aux/print-committerlist.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/sh
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+function get_contributors {
+  local OS=`uname -s`
+  case "$OS" in
+  Linux|CYGWIN*) # GNU sed
+    local SED_ERE_FLAG=-r
+  ;;
+  *) # BSD sed
+    local SED_ERE_FLAG=-E
+  ;;
+  esac
+
+  local CONTRIB_EMAIL_SED_COMMAND="s/^[[:blank:]]{5}[[:digit:]]+[[:blank:]]/ * /"
+  if [ "$1" == "couchdb-main-repo" ]
+  then
+    git shortlog -se 6c976bd..HEAD \
```

Comment:
it is based on the current 1.x process which i made work for our subrepos/2.0 - for now i would like to keep that, but feel free to raise that discussion topic on the ML

(see https://github.com/apache/couchdb/blob/1.6.x/bootstrap#L57-63)